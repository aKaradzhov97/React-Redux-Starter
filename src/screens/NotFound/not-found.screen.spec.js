import React from 'react';
import NotFoundScreen from './not-found.screen';
import { render } from '../../tests/test-utils';

describe('NotFoundScreen', () => {
  let initialState;

  beforeAll(() => {
    initialState = {};
  });

  describe('Rendering', () => {
    it('should render the component with initial state', () => {
      const { container } = render(<NotFoundScreen />, { initialState });

      expect(container.querySelector('body section')).toMatchSnapshot();
    });
  });
});
