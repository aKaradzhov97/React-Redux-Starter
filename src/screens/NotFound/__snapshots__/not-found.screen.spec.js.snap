// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen Rendering should render the component with initial state 1`] = `
<section>
  <h1>
    not_found_screen.heading
  </h1>
  <p>
    not_found_screen.paragraph
  </p>
</section>
`;
